<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="5,2">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QWebEngineView" name="webEngineView">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,1,1,1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Accident Predictions</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="t1_accident">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>1. 사고유형1</string>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="t2_accident">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2. 사고유형 2</string>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="t3_accident">
         <property name="font">
          <font>
           <family>맑은 고딕</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>3. 사고유형 3</string>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Information Input</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="lat_label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Latitude</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignHCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="lat_input">
             <property name="decimals">
              <number>10</number>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>35.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="lon_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Longitude</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignHCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="lon_input">
             <property name="decimals">
              <number>10</number>
             </property>
             <property name="maximum">
              <double>359.999999000000003</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>129.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Weather</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="weather_comboBox">
           <property name="currentText">
            <string>양호</string>
           </property>
           <item>
            <property name="text">
             <string>양호</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>황천2급</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>황천3급</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>황천4급</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>황천5급</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>황천6급</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>저시정</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>태풍경보</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>태풍주의보</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>풍랑경보</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>풍랑주의보</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Vessel Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="vessel_comboBox">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>어선</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <item>
            <property name="text">
             <string>어선</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>여객선</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>화물선</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>유조선</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>고무보트</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>관공선</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>낚시어선</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>모터보트</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>예부선</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>요트</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>유도선</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="update_button">
         <property name="text">
          <string>Update Information</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
